src/client/src/index.js

```diff
@@ -18,13 +18,35 @@ const CaptionForm = require('./views/CaptionForm')
 
 m.route(document.body, '/start', {
   '/start': Start,
-  '/scenes': Scenes,
+  '/scenes': {
+    onmatch: function() {
+      if(Scene.getInputFormat() === 'Select your option') {
+        alert('Must specify caption format')
+        m.route.set('/start')
+      }
+      else {
+        return Scenes
+      }
+    }
+  },
   '/scenes/scene-:sceneId': {
+    onmatch: function() {
+      if(Scene.getInputFormat() === 'Select your option') {
+        alert('Must specify caption format')
+        m.route.set('/start')
+      }
+    },
     render: function(vnode) {
       return m(SceneForm, vnode.attrs)
     }
   },
   '/scenes/scene-:sceneId/caption-:captionId': {
+    onmatch: function() {
```

Comment:
There's a lot of duplicate code with these onmatch functions. I would like to see this refactored so we aren't repeating so much code.

Otherwise looks good.

Reply:
Done